src/tests_package.rs

```diff
@@ -19,10 +21,11 @@ static INTEGRATION_TESTS_RS: Lazy<PathBuf> =
 
 pub static TEST_DEPENDENCIES: Lazy<String> = Lazy::new(|| {
     format!(
-        "{}{}{}",
+        "{}{}{}{}",
         CL_CONTRACT.display_with_features(false, vec!["test-support"]),
         CL_ENGINE_TEST_SUPPORT.display_with_features(true, vec!["test-support"]),
         CL_TYPES.display_with_features(true, vec![]),
+        CL_EXECUTION_ENGINE.display_with_features(true, vec![])
```

Comment:
Please move this up one line so the generated Cargo.toml has alphabetically-ordered deps.